sdclientapi/__init__.py

```diff
@@ -480,7 +562,9 @@ def download_submission(
         :returns: Tuple of sha256sum and path of the saved submission.
         """
         path_query = "api/v1/sources/{}/submissions/{}/download".format(
-            submission.source_uuid, submission.uuid
+            submission.source_uuid,
+            submission.uuid,
+            timeout=timeout or self.default_download_timeout,
```

Comment:
Does this work? Won't `timeout` get ignored since there isn't a `{timeout}` var in the string?

Reply:
Also do we want to pass a timeout to `download_reply` the way we do here for `download_submission`?

Reply:
Oh wow, I added that to the wrong place entirely. Good catch.

Reply:
Also I figured it's ok for now to not pass this to replies since realistically we aren't going to have huge replies.